PowerEditor/src/Parameters.cpp

```diff
@@ -5096,7 +5096,16 @@ void NppParameters::feedGUIParameters(TiXmlNode *node)
 			const TCHAR * optDocPeekOnMap = element->Attribute(TEXT("docPeekOnMap"));
 			if (optDocPeekOnMap)
 				_nppGUI._isDocPeekOnMap = (lstrcmp(optDocPeekOnMap, TEXT("yes")) == 0);
-
+		}
+		else if (!lstrcmp(nm, TEXT("CommandLineInterpreter")))

```

Comment:
The naming should follow the convention.

Reply:
Name of setting changed from **`CommandLineInterpreter`** to **`commandLineInterpreter`** (also in commit message).